.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/psf/black
-    rev: stable
+    rev: 20.8b1
```

Comment:
Er, maybe this is a mistake. I was getting conflicts between the pre-commit version and the locally installed version once I updated, but [the Black docs say that](https://black.readthedocs.io/en/stable/version_control_integration.html)

> `stable` is a branch that tracks the latest release on PyPI

So I'm not sure why I'm getting a conflict with it then.